make.js

```diff
@@ -41,11 +42,19 @@ target.build = function() {
 
 target.units = function() {
     target.build();
+    var nodeVer = process.versions.node;
+    if(semver.lt(nodeVer,'8.0.0')){
+        pushd('_build');
```

Comment:
Nit: could we set pushd parameter conditionally only - and leave other commands as is?

Reply:
yeah, seems like a good idea